release_notes/48-release-notes.adoc

```diff
@@ -365,4 +365,11 @@ As a result, the Scanner V4 now accurately identifies critical CVEs in Java work
 This lack of feedback occurred because the button only reset the form for unpersisted changes.
 This update introduces an **Edit** button to initiate editing, making the **Save** and **Cancel** buttons visible and enabled only when you make changes.
 
-include::modules/image-versions.adoc[leveloffset=+1]
\ No newline at end of file
+[id="upcoming-changes_49"]
+== Upcoming admission controller enforcement changes in version 4.9.0
+
+{product-title-short} 4.9 streamlines admission controller configuration by consolidating the existing `listen` and `enforce` settings into a single **Enforcement** option.
+
+You can select `Yes` to enable enforcement for create, update, and scale events. Or, select `No` to disable enforcement for these events.
```

Comment:
The can select the following settings for the **Enforcement** option:

* `Yes` to enable enforcement for create, update, and scale events.
* `No` to disable enforcement for events.